source/Audio.cpp

```diff
@@ -0,0 +1,71 @@
+#include "Audio.hpp"
+#include "Native.hpp"
+#include "Entity.hpp"
+
+namespace GTA
+{
+	int Audio::GetSoundID()
+	{
+		return Native::Function::Call<int>(Native::Hash::GET_SOUND_ID);
```

Comment:
Any info on how to use this? How do we get the ID of sounds which other methods use? (like HasSoundFinished)